liferay-tango/liferay-portal
# LPS-133354 Mime Types filter

Motivation
=======
[Link to story](https://issues.liferay.com/browse/LPS-133354)

Proposed Solution
========
Developed new feature for filtering content by file extension in Content Dashboard.

Change summary:
---------------
* TODO

Steps to Verify:
----------------
1. Add some documents and images to site: Site menu -> Content & Data -> Documents and media
2. Go to content dashboard and start filtering the data in the table by file extension and check that everything works as expected.

Screenshots (optional):
-----------------------
<img width="1237" alt="Captura de pantalla 2021-08-18 a las 16 30 54" src="https://user-images.githubusercontent.com/3276218/129917161-66a53de9-e509-4eba-8ac3-64e957b303c0.png">
<img width="1724" alt="Captura de pantalla 2021-08-18 a las 16 31 24" src="https://user-images.githubusercontent.com/3276218/129917183-96d28186-67d5-401a-8d4f-32efc20a8c66.png">
<img width="1244" alt="Captura de pantalla 2021-08-18 a las 16 31 48" src="https://user-images.githubusercontent.com/3276218/129917202-ac537e8f-7087-4daf-a590-31ea7546f934.png">
